docs/sources/flow/reference/components/otelcol.processor.probabilistic_sampler.md

```diff
@@ -0,0 +1,139 @@
+---
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.processor.probabilistic_sampler/
+labels:
+  stage: opensource
```

Comment:
Normally the value of "stage" will be one of the three values listed in the [Stability docs](https://grafana.com/docs/agent/latest/stability/):

- Experimental
- Beta
- Stable

`probabilistic_sampler` has different stabilities for logs (alpha) and traces (beta), so we should make that clear too.

Unfortunately, when I added the last few Otel Flow components, I forgot to set the stability labels on them. There is an ongoing discussion in these two PRs:
https://github.com/grafana/agent/pull/4948
https://github.com/grafana/agent/pull/4343

The main part is that we should not label something as less than "stable" if we have already listed it as "stable" in previous versions. So I'd suggest labeling `probabilistic_sampler` as `Experimental`, and we can change it later if it's more appropriate.